packages/big-design/src/components/Datepicker/Header.tsx

```diff
@@ -0,0 +1,48 @@
+import { ChevronLeftIcon, ChevronRightIcon } from '@bigcommerce/big-design-icons';
+import React from 'react';
+
+import { Flex } from '..';
+import { Button } from '../Button';
+import { Text } from '../Typography';
+
+export interface HeaderProps {
+  date?: Date;
+  nextMonthButtonDisabled?: boolean;
+  prevMonthButtonDisabled?: boolean;
+  decreaseMonth?(): void;
+  increaseMonth?(): void;
+}
+
+const months: string[] = Array(12)
+  .fill('')
+  .map((_, i) => new Date(0, i).toLocaleString('en', { month: 'long' }));
+
+const Header: React.FC<HeaderProps> = ({
+  date = new Date(),
+  decreaseMonth,
+  increaseMonth,
+  prevMonthButtonDisabled,
+  nextMonthButtonDisabled,
+}) => (
+  <Flex alignItems="center" justifyContent="space-between">
+    <Button
+      type="button"
+      iconOnly={<ChevronLeftIcon />}
```

Comment:
This should have a `title` as well. Also the title should reflect the action you are taking. So like `View previous month.` (Notice the `.` at the end, this helps create a pause for screen readers).

Reply:
Doesn't look like this was resolved. `title` should be on the Icon as a prop.